packages/examples/packages/manage-state/src/utils.ts

```diff
@@ -61,14 +76,21 @@ export async function setState(newState: State) {
  *
  * This uses the `snap_manageState` JSON-RPC method to clear the state.
  *
+ * @param encrypted - An optional flag to indicate whether to use encrypted storage or not.
  * @see https://docs.metamask.io/snaps/reference/rpc-api/#snap_managestate
  */
-export async function clearState() {
+export async function clearState(encrypted?: boolean) {
   await snap.request({
     method: 'snap_manageState',
 
     // For this particular example, we use the `ManageStateOperation.ClearState`
     // enum value, but you can also use the string value `'clear'` instead.
-    params: { operation: ManageStateOperation.ClearState },
+    params: {
+      operation: ManageStateOperation.ClearState,
+      // By default all state is encrypted, but you can choose to not encrypt it
+      // To do this you may set this flag to false
+      // This will use a separate unencrypted storage from the encrypted state
```

Comment:
```suggestion
      // By default all state is encrypted, but you can choose to not encrypt it.
      // To do this you may set this flag to false.
      // This will use a separate unencrypted storage from the encrypted state.
```